apps/address-service/domains/address/schema/AddressSource.js

```diff
@@ -14,7 +14,7 @@ const { SOURCE_ALREADY_EXISTS_ERROR } = require('@address-service/domains/addres
 const { AddressSource: AddressSourceApi } = require('@address-service/domains/address/utils/serverSchema')
 
 
-const errors = {
+const ERRORS = {
```

Comment:
It's not a part of this PR. You should change it everywhere If you want. Rather than change it in some files.

Reply:
I've refactored it everywhere except submdules